### PR TITLE
add skd

### PR DESCRIPTION
**Pull request by user3984**
add skd loss

### CODE REVIEW

File: ppcls/configs/ImageNet/Distillation/resnet34_distill_resnet18_skd.yaml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        weight: 1.0
+        
+
+Optimizer:
+  name: Momentum
+  momentum: 0.9
+  weight_decay: 1e-4
+  lr:
+    name: MultiStepDecay
+    learning_rate: 0.2
```
Comment (littletomatodonkey) [inline ppcls/configs/ImageNet/Distillation/resnet34_distill_resnet18_skd.yaml]: 改成0.1

File: ppcls/configs/ImageNet/Distillation/resnet34_distill_resnet18_skd.yaml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+              flip_code: 1
+          - NormalizeImage:
+              scale: 0.00392157
+              mean: [0.485, 0.456, 0.406]
+              std: [0.229, 0.224, 0.225]
+              order: ''
+
+    sampler:
+        name: DistributedBatchSampler
+        batch_size: 128
```
Comment (littletomatodonkey) [inline ppcls/configs/ImageNet/Distillation/resnet34_distill_resnet18_skd.yaml]: 改成64，和其他蒸馏算法的配置保持一致

File: docs/zh_CN/training/advanced/knowledge_distillation.md [hunk 3]
```diff
@@ -654,6 +655,72 @@ Loss:
...
+
+| 策略 | 骨干网络 | 配置文件 | Top-1 acc | 下载链接 |
+| --- | --- | --- | --- | --- |
+| baseline | ResNet18 | [ResNet18.yaml](../../../../ppcls/configs/ImageNet/ResNet/ResNet18.yaml) | 70.8% | - |
+| SKD | ResNet18 | [resnet34_distill_resnet18_skd.yaml](../../../../ppcls/configs/ImageNet/Distillation/resnet34_distill_resnet18_skd.yaml) | 72.84%(**+2.04%**) | - |
+
+
+##### 1.2.9.2 SKD 配置
+
+SKD 配置如下所示。在模型构建Arch字段中，需要同时定义学生模型与教师模型，教师模型固定参数，且需要加载预训练模型。在损失函数Loss字段中，需要定义`DistillationSKDLoss`（学生与教师之间的SKD loss），作为训练的损失函数。
```
Comment (littletomatodonkey) [inline docs/zh_CN/training/advanced/knowledge_distillation.md]: 再简单介绍下，这个loss包含了学生与教师模型之间的kl div loss与学生模型与真值之间的gt loss

File: docs/en/advanced_tutorials/knowledge_distillation_en.md [hunk 4]
```diff
@@ -655,6 +656,72 @@ Loss:
...
+> 2022, under review
+
+Due to the limited capacity of the student, student performance would unexpectedly drop when distilling from an oversized teacher. Spherical Knowledge Distillation (SKD) explicitly eliminates the gap of confidence between teacher and student, so as to ease the capacity gap problem. SKD achieves a significant improvement over previous SOTA in distilling ResNet18 on ImageNet1k.
+
+Performance on ImageNet1k is shown below.
+
+| Strategy | Backbone | Config | Top-1 acc | Download Link |
+| --- | --- | --- | --- | --- |
+| baseline | ResNet18 | [ResNet18.yaml](../../../../ppcls/configs/ImageNet/ResNet/ResNet18.yaml) | 70.8% | - |
+| SKD | ResNet18 | [resnet34_distill_resnet18_skd.yaml](../../../../ppcls/configs/ImageNet/Distillation/resnet34_distill_resnet18_skd.yaml) | 72.84%(**+2.04%**) | - |
```
Comment (cuicheng01) [inline docs/en/advanced_tutorials/knowledge_distillation_en.md]: 两个链接失效，check下吧

**Review by cuicheng01 — APPROVED** [review]
